exercises/collatz-conjecture/metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Calculate the number of steps to reach 1 given an integer using Collatz Conjecture"
```

Comment:
> "given an integer" makes this sound a bit confusing to me

same, because it makes it sound like the integer is using the Collatz conjecture